src/matlab.ts

```diff
@@ -50,17 +58,23 @@ export async function getVersion(release: string): Promise<Version> {
         return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));
     }
 
-    let parsedRelease: string = release.toLowerCase();
+    let parsedRelease: string = release.toLowerCase().trimStart().trimEnd();
```

Comment:
I think there's a String.trim() in JS, no?
```suggestion
    let parsedRelease: string = release.toLowerCase().trim();
```